tox.ini

```diff
@@ -61,7 +50,13 @@ commands =
 
 [flake8]
 extend-ignore = 
-    E203,
+    E203, # whitespace before ':' 
+    E501, # line too long
+    F401, # imported but unused
```

Comment:
this seems like a useful one to have so that we don't have extra imports, is it important to ignore it?

Reply:
this refers to `F401 # imported but unused`

Reply:
Good point, fixed these issues and remove the ignore